### DISCUSSION TITLE
Rest Of Search Placement Bid Adjustment Calculation

### DISCUSSION THREAD

**Discussion by Lin1031** [amzn/ads-advanced-tools-docs#172] [Q&A] [answered]
How to calculate the highest bid of a SP campaign when adjusting bid for rest of search placement to 100% and using Dynamic bids - up and down strategy？For example, a keyword's basic bid is 1. Please show me the formula. thanks

**Comment by jerirose** [comment, marked as answer]
Hi @Lin1031 , you can find information on dynamic bidding in the [Amazon Ads Support Center page on bidding strategies](https://advertising.amazon.com/help/GCU2BUWJH2W3A8Z7). Here's the relevant passage:

>### Dynamic bids - up and down
>
>With dynamic bids - up and down' strategy, Amazon will increase your bids in for clicks that may be more likely to convert to a sale. With this strategy, we’ll also reduce your bids for clicks that are less likely to convert to a sale.
>
>We won’t increase your bids by more than 100% for placements at the top of the first page of search results, nor more than 50% for all other placements. Take this into account when selecting your bid while using this strategy. Since this strategy adjusts your bid up and down in proportion to likelihood of a conversion, it may deliver more conversions for your ad spend compared to the other 2 strategies. If we consider your ad is more likely to convert to a sale, we might increase your bid for that auction. This could happen, for example, if your ad appears for a highly relevant search query or on a placement that performs well. If we notice another opportunity that looks less likely to convert to a sale, we might lower your bid for that auction. For example, Amazon can adjust your bid of $1.00 up to $2.00 for opportunities on top of the first page of search results, and up to $1.50 for opportunities on all other placements.
>
>Note: With both dynamic bids - up and down, and dynamic bids - down only strategies, we'll adjust your bids in real time.
